### PR TITLE
Added reviews and reports

### PR DESCRIPTION
**Pull request by uygaraktuna**

### CODE REVIEW

File: src/main/java/com/latebox/server/report/ReportRepository.java [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.stereotype.Repository;
+
+import java.util.List;
+
+@Repository
+public interface ReportRepository extends JpaRepository<Report, Long> {
+
+
+    List<Report> findByName(String name);
```
Comment (tahelmo) [inline src/main/java/com/latebox/server/report/ReportRepository.java:12]: Report entity does not have a name property. This line need to be removed.

File: src/main/java/com/latebox/server/review/ReviewRepository.java [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.stereotype.Repository;
+
+import java.util.List;
+
+@Repository
+public interface ReviewRepository extends JpaRepository<Review, Long> {
+
+
+    List<Review> findByName(String name);
```
Comment (tahelmo) [inline src/main/java/com/latebox/server/review/ReviewRepository.java:12]: Report entity does not have a name property. This line need to be removed.

File: src/main/java/com/latebox/server/report/ReportRepository.java (hunk 1 above)
Comment (uygaraktuna) [reply src/main/java/com/latebox/server/report/ReportRepository.java:12]: Thanks

**Review by tahelmo — APPROVED** [review]
Looks OK to be merged

**Comment by uygaraktuna** [comment]
bugs fixed, merging

**Pull request merged**